src/arithmetic/algebraic_expression.h

```diff
@@ -66,11 +67,17 @@ typedef struct {
     Node *src_node;                         // Nodes represented by the first operand columns.
     Node *dest_node;                        // Nodes represented by the last operand rows.
     Edge *edge;                             // Edge represented by sole operand.
-    AST_LinkLength *edgeLength;             // Repeatable edge length.
 } AlgebraicExpression;
 
-/* Construct an algebraic expression from a query. */
-AlgebraicExpression **AlgebraicExpression_From_Query(const AST *ast, Vector *matchPattern, const QueryGraph *q, size_t *exp_count);
+/* Constructs an empty expression. */
+AlgebraicExpression *AlgebraicExpression_Empty();
```

Comment:
Safer to do:
`AlgebraicExpression *AlgebraicExpression_Empty(void);`

Reply:
Ok, Whys that safer?

Reply:
Empty parentheses in a function declaration allow that function to accept an unbounded number of arguments, while `(void)` specifies that the function will accept 0 arguments.

Reply:
👍